packages/vaadin-lumo-styles/badge.js

```diff
@@ -86,31 +86,36 @@ const badge = css`
 
   /* Icon */
 
+  [theme~='badge'] vaadin-icon,
   [theme~='badge'] iron-icon {
     margin: -0.25em 0;
     --iron-icon-width: 1.5em;
     --iron-icon-height: 1.5em;
   }
 
+  [theme~='badge'] vaadin-icon:first-child,
   [theme~='badge'] iron-icon:first-child {
     margin-left: -0.375em;
   }
 
+  [theme~='badge'] vaadin-icon:last-child,
   [theme~='badge'] iron-icon:last-child {
     margin-right: -0.375em;
   }
 
-  [theme~='badge'][icon] {
+  iron-icon[theme~='badge'][icon],
+  vaadin-icon[theme~='badge'][icon] {
     min-width: 0;
     padding: 0;
     font-size: 1rem;
-    --iron-icon-width: var(--lumo-icon-size-m);
-    --iron-icon-height: var(--lumo-icon-size-m);
+    width: var(--lumo-icon-size-m);
+    height: var(--lumo-icon-size-m);
```

Comment:
`vaadin-icon` doens't have such a CSS variable but I assumed the intention is to affect the size of the same component that has `theme="badge" icon=...`